releasenotes/notes/34801.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: release-notes/v2
```

Comment:
Since this release notes file is empty and not needed for this PR as it sits now, this file can be removed and the `release-notes-none` label set on the PR

Reply:
Fine,thanks a lot. But I don't find the way to set the label ```release-notes-none``` , is it open to contributors ?